src/modules/Microsoft.FinOpsHubs/linkedservices/deploy.bicep

```diff
@@ -0,0 +1,75 @@
+// Source: 
+// Date: 2023-02-02
+// Version: 
+
+@description('Conditional. The name of the parent Azure Data Factory. Required if the template is used in a standalone deployment.')
+param dataFactoryName string
+
+@description('The name of the service to link to.')
+param linkedServiceName string
+
+@allowed([
+  'AzureBlobFS'
+  'AzureKeyVault'
+])
+@description('Required. The type of Linked Service.')
+param linkedServiceType string
+
+@description('The name of the service to link to.')
+param storageAccountName string = 'none'
+
+@description('The name of the service to link to.')
+param keyVaultName string
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
```

Comment:
nit: Remove if it's not a CARML module. We don't need this for ours.

Reply:
Done.